.circleci/config.yml

```diff
@@ -19,10 +19,15 @@ executors:
   shellcheck:
     docker:
       - image: koalaman/shellcheck-alpine@sha256:dfaf08fab58c158549d3be64fb101c626abc5f16f341b569092577ae207db199
+  playwright:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.39.0-jammy
 
 orbs:
   gh: circleci/github-cli@2.0
   codecov: codecov/codecov@3.2.2
+  slack: circleci/slack@4.12.5
+
```

Comment:
I'm assuming this is to announce in a slack channel about test status?

Reply:
yes, correct